modules/evm/src/runner/native.rs

```diff
@@ -107,9 +119,16 @@ impl<T: Trait> RunnerT<T> for Runner<T> {
 				logs,
 			};
 
+			debug::debug!(
+				target: "evm",
+				"create-result: create_info {:?}",
+				create_info
+			);
+
 			match reason {
 				ExitReason::Succeed(_) => match substate.gasometer.record_deposit(out.len()) {
 					Ok(()) => {
+						substate.inc_nonce(address);
```

Comment:
@xlc why incrementing contract nonce?

Reply:
To confirm eth 161 https://github.com/ethereum/EIPs/blob/master/EIPS/eip-161.md